cluster/images/hyperkube/Makefile

```diff
@@ -74,6 +76,8 @@ endif
 ifeq ($(ARCH),amd64)
 	# When building "normally" for amd64, remove the whole line, it has no part in the amd64 image
 	cd ${TEMP_DIR} && ${SED_CMD} "/CROSS_BUILD_/d" Dockerfile
+else ifeq ($(ARCH),ppc64le)
```

Comment:
This will break the release scripts, as ppc64le must be cross-built when on amd64.
What we should do instead is check the host platform and the target platform. If they equal,
```
cd ${TEMP_DIR} && ${SED_CMD} "/CROSS_BUILD_/d" Dockerfile
```
should be run.
If they aren't the same, the second case should be run.

So add a `HOSTARCH?=amd64` variable and compare `ifeq($(ARCH),$(HOSTARCH))` above.
Then you set `HOSTARCH=ppc64le` when running the script, and it will work